ocs_ci/ocs/machine.py

```diff
@@ -138,6 +140,63 @@ def get_machinesets():
     return machine_sets
 
 
+def get_machine_from_machineset(machine_set):
+    """
+    Get the machine name from its associated machineset
+
+    Args:
+        machine_set (str): Name of the machine set
+
+    Returns:
+        List: Machine names
+    """
+    machine_objs = get_machine_objs()
+    machine_set_list = []
+    for machine in machine_objs:
+        if machine.get().get(
+                'metadata'
+        ).get('name')[:-6] == machine_set:
+            return machine_set_list.append(
```

Comment:
that's strange.
It will return on the first element that complies with the 'if' statement

Reply:
Done. Came up with a new logic for this.